tests/test_client_parameters.py

```diff
@@ -29,6 +29,20 @@
             AwsClientParameters(api_version="1.0.0"),
             {"api_version": "1.0.0"},
         ),
+        # would need to implement a custom equality function or class matcher,
+        # but this is too trivial of a test
+        # (
+        #     AwsClientParameters(config=botocore.client.Config(
+        #         region_name="eu_west_1",
+        #         retries={"max_attempts": 10, "mode": "standard"}
+        #     )),
+        #     {
+        #         "config": {
+        #             "region_name": "eu_west_1",
+        #             "retries": {"max_attempts": 10, "mode": "standard"},
+        #         }
+        #     },
+        # ),
```

Comment:
I think this can be uncommented since it tests that you can pass a config kwarg.

Reply:
But the test ultimately fails, thus my explanatory comment - I thought adding in the necessary code to check equality of botocore.client.Config (it isn't a dict) to the dict would be kind of out of scope. I am not a pytest expert so maybe there is an easier way?  I added the whole comment to clearly state this simple test had not been overlooked given the other ones.

Or perhaps add some pytest code to mark the test as xfail?  Not sure what the Prefect testing philosophy is

Reply:
ok, I have enabled the test and set it to xfail